src/darwin/Framework/CHIP/MTRDeviceController_Internal.h

```diff
@@ -72,6 +72,10 @@ NS_ASSUME_NONNULL_BEGIN
 // (moved here so subclasses can initialize differently)
 @property (readwrite, retain) dispatch_queue_t chipWorkQueue;
 
+// Counters to track assertion status
```

Comment:
Should document what synchronizes access to these, whether it's "must only touch on Matter queue" or some sort of locking.